src/util/ws_message.py

```diff
@@ -13,7 +13,23 @@
 #           }
 
 
-def format_response(incoming_msg: Dict[str, Any], response_data: Dict[str, Any]) -> str:
+Request = TypedDict(
+    "Request", {"command": str, "data": Optional[Dict[str, Any]], "request_id": str, "destination": str, "origin": str}
+)
+Response = TypedDict(
+    "Response",
+    {
+        "command": str,
+        "ack": bool,
+        "data": Optional[Dict[str, Any]],
```

Comment:
I'm not sure about the `Optional` in the response.

Reply:
Is there any place where it's None? probably better to not make it optional

Reply:
Done